app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -220,6 +231,22 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType = Content
                         required={!!props.values.otherSource}
                       />
                     </Row>
+                    <Row>
+                      <FormikText
```

Comment:
Is this a new field or is it just moved?

Reply:
Just moved, also regarding the two content forms. I think I am going to take another shot at controlling it with a `condensed` prop and some conditional rendering - working on that right now

Reply:
Nevermind I think it is more confusing with that...